### DISCUSSION TITLE
client on Mac

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1134] [Q&A]
Hi, so in the meantimes I can't use the VM client in the MAC (M1) laptop right ? Because I want to use client to set the loco position system anchors and through the other tools to know the crazyflie data and state. These are currently not possible right ? I followed the instruction(https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/#mac)but when I doing the second step, it will stuck.
(arch --x86_64 brew install python@3.9 libusb) this command.

**Comment by ataffanel** [comment]
How does it get stuck? what error is displayed?

One thing to try would be to bump-up the python version `python@3.11` for example

**Reply by CalvinKoung** [reply]
It doesn't appear any error. Just no reponse in terminal.

**Reply by ataffanel** [reply]
What version of macos are you using, is brew working for any other packages?

**Reply by CalvinKoung** [reply]
Sonoma 14.2.1
Yes, it normal working other brew command.

**Reply by ataffanel** [reply]
The fact that bew does not show anything is strange, it might be a problem with your brew install. On my side brew does start to install python but it tries to compile it which takes ages. There is a way to force brew to install recompiled packages and that works:
``` bash
brew install --force-bottle python@3.11
```

However, while ```pip3.11 install cfclient``` works, there has been a change in Mac Sonoma that makes the client not starting (they removed opengl or something like that and so some libs are not found). Unfortunatly this is something that will take more time to fix, so for now and until this bug is fixed the client does not support mac on Sonoma :-(. I created an [issue on the client repos.](https://github.com/bitcraze/crazyflie-clients-python/issues/694) In the way forward would be to use another computer or to try to fix the bug.

**Reply by CalvinKoung** [reply]
ok, thanks

**Reply by knmcguire** [reply]
A fix is proposed for apple M1 opengl / vispy problem in https://github.com/bitcraze/crazyflie-clients-python/issues/694
